build.gradle.kts

```diff
@@ -50,6 +50,26 @@ subprojects {
     }
 }
 
+tasks.register("updateReadme") {
+    val filename = "README.md"
+    val kmpRegex = Regex("io\\.sentry:sentry-kotlin-multiplatform:[^\"]*")
+    val cocoaRegex = Regex("pod\\(\"Sentry\", \"~>[^\"]*\"\\)")
+
+    doLast {
+        try {
+            val readmeFile = File(filename)
+
+            val newContents = readmeFile.readText()
+                .replace(kmpRegex, "io.sentry:sentry-kotlin-multiplatform:${project.version}")
+                .replace(cocoaRegex, "pod(\"Sentry\", \"~> ${Config.Libs.sentryCocoaVersion}\")")
```

Comment:
The project version isn't necessarily published yet.
I believe that using the maven badges with the latest version (similar to sentry-java) and using a placeholder in the versioning is more effective.

Reply:
Maybe this could only be run on release branches? This would be useful for managing the required sentry cocoa version otherwise we'd need to manage that manually in the README

Reply:
Sounds good - I'd like to at least keep the required sentry cocoa sdk version documented for the user

Reply:
As sentry-docs needs to be in sync with cocoa version as well, we should do it manually for now https://github.com/getsentry/sentry-docs/pull/6452#discussion_r1136921094